koku/api/provider/view.py

```diff
@@ -130,6 +130,9 @@ def get_queryset(self):
         by filtering against a `user` object in the request.
         """
         queryset = Provider.objects.none()
+        stat = self.request.query_params.get('stats', None)
```

Comment:
`None` is the default, if you don't provide a default. You don't need to explicitly state it here.
`stat = self.request.query_params.get('stats')`